packages/fxa-shared/db/index.ts

```diff
@@ -64,6 +64,9 @@ export function setupDatabase(opts: MySQLConfig): Knex {
     client: 'mysql',
     pool: {
       afterCreate: setupConnection,
+      min: 0,
```

Comment:
Might be best to set this to a portion of the max (1/5th or so), to ensure there's idle connections around.